src/PanZoom.js

```diff
@@ -240,15 +310,52 @@ class PanZoom extends React.Component<Props, State> {
   }
 
   onWheel = (e: WheelEvent) => {
-    const { disableScrollZoom, disabled, zoomSpeed } = this.props
-    if (disableScrollZoom || disabled) {
-      return
-    }
+    const { disableScrollZoom, disabled, zoomSpeed, noStateUpdate, hasNaturalScroll } = this.props
 
-    const scale = getScaleMultiplier(e.deltaY, zoomSpeed)
-    const offset = this.getOffset(e)
-    this.zoomTo(offset.x, offset.y, scale)
-    e.preventDefault()
+    if (disabled) return
+
+    if (!disableScrollZoom && this.ctrlKeyPressed) {
+      const scale = getScaleMultiplier(e.deltaY, zoomSpeed)
+      const offset = this.getOffset(e)
+      this.zoomTo(offset.x, offset.y, scale)
+      e.preventDefault()
+    } else {
+      if (e.cancelable) {
```

Comment:
This second section handles adapting wheel events into this components pan lifecycle. Basically a pan begins when the first wheel event enters the system, more wheel events will continue the pan, and a timeout will wait for the end of wheel events before ending the pan lifecycle.